cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -10,6 +10,16 @@ KUBECONFIG_PATH="${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig"
 MASTER_NODE="${CLUSTER_NAME}-control-plane"
 WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
+if ! skopeo -v &> /dev/null
+then
+        echo "skopeo could not be found, creating alias for sriov operator, see hack/env.sh"
+        cat <<EOF > /usr/local/bin/skopeo
```

Comment:
By whom this script is ran? From within a container?
I'm asking because a regular user does not have permissions to create files there.

I also do not understand how `quay.io/skopeo/stable` is related to the sriov operator. To me it looks like you use a skopeo container instead of the binary.

Reply:
I dont use sudo for that, please see that i create alias that use the container as you said,
/usr/loca/bin without installing anything.

It runs here, 
https://github.com/openshift/sriov-network-operator/blob/release-4.6/hack/env.sh#L1
which runs in the same level config_sriov.sh runs, which is the level where cluster-up runs
this answer how its related to sriov operator (even 4.4 runs skopeo, and 4.4 is the one we have now, which doesnt have the `exit 1` yet)

Reply:
config_sriov.sh runs from cluster-up/kind-k8s-sriov-1.17.0/provider.sh
hack/env/sh is sriov-operator script https://github.com/openshift/sriov-network-operator/blob/release-4.4/hack/env.sh
Skopeo is being used by the sriov-opeator build scripts.

Reply:
If I understand correctly, you add a script in `/usr/local/bin` to run the container, right? But adding a script there requires root as far as I know.

Reply:
you right we need sudo for that, 
for 4.6 we must have skopeo, so everyone that will run the provider once we bump will fail due to
https://github.com/openshift/sriov-network-operator/blob/release-4.6/hack/env.sh#L1

and the code that i added, tries to create, if it manage all good, and it does manage on CI, i just checked
if it doesnt (the code in this PR), it doesnt fail, but let the sriov fail himself on 4.6, and then the user will need to install himself
so all will be good imo, wdyt?

Reply:
i can add || true to the commands , since there is -e

Reply:
we can also ask SRIOV team to skip need of skopeo if all the env-vars exists,
but not sure they will backport it to 4.6 or even do it, but it might make sense to do it
looking on
https://github.com/openshift/sriov-network-operator/blob/release-4.6/hack/env.sh#L1

Reply:
not ortogonal to this PR, since we would still be able to remove our config from config_sriov.sh maybe,
but suggested a PR

https://github.com/openshift/sriov-network-operator/pull/396

Reply:
https://github.com/openshift/sriov-network-operator/pull/396

Reply:
https://github.com/openshift/sriov-network-operator/pull/396 is merged,
in case it would be backported or in case we would bump to a version which has this feature, we would be able to use SKIP_VAR_SET, 
and continue current approach we have, unless we adapt a better one (it should be one liner imo, mentioning the release only, once its possible)

Reply:
https://github.com/kubevirt/kubevirtci/pull/461